src/main/java/com/service/PersonService.java

```diff
@@ -93,4 +97,24 @@ public void showPersons() {
             System.out.println(person.toString());
         }
     }
+
+    @Override
+    public void showPersonsFromFile() {
+
+        String fileLoc = "C:\\Петко\\udemy\\java master class\\Martin_Project\\personsList.txt";
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileLoc))) {
+
+            String personLine;
+
+            while ((personLine = bufferedReader.readLine()) != null) {
+                String[] data = personLine.split(",");
+                UUID uuid = UUID.fromString(data[0]);
+                String name = data[1];
+                System.out.println(uuid + ", " + name);
```

Comment:
Show the whole person, not only the name here. And stop using pluses in strings use String.format